docs/manage-manually.md

```diff
@@ -0,0 +1,318 @@
+# Manual management of database clusters deployed with Percona Operator for PostgreSQL
+
+The purpose of the Operator is to automate database management tasks for you. However, you may need to manage the database cluster manually. For example, to troubleshoot issues or for maintenance. 
+
+The following sections explain how you can manage your cluster manually. 
+
+## Disable health check probes for maintenance
+
+Probes are tasks Kubernetes runs to gather information about the health and status of containers running within Pods. They serve as a mechanism to ensure the system is running smoothly by periodically checking the state of applications and services.
+
+Kubernetes has various types of probes:
+
+* Startup probe verifies whether the application within a container is started
+* Liveness probe determines when to restart a Pod
+* Readiness probe checks that the container is ready to start accepting traffic
+
+Sometimes it's necessary to take a manual control over the `postgres` process for maintenance. This means you need to disable a Kubernetes liveness probe so that it doesn't restart the database container during the maintenance period.
+
+Here's what you need to do:
+
+1. Create a `sleep-forever` file in the data directory with the following command:
+
+    ```{.bash data-prompt="$"}
+    $ kubectl exec cluster1-instance1-24b8-0 -- touch /pgdata/sleep-forever
+    ```
+
+2. Optionally, you can delete the Pod:
```

Comment:
not optional, you should delete the pod :D

Reply:
Updated, removed Optional part